ui/ozone/platform/starboard/platform_event_source_starboard.cc

```diff
@@ -308,6 +308,29 @@ void PlatformEventSourceStarboard::HandleEvent(const SbEvent* event) {
       FROM_HERE, base::BindOnce(&DeliverEventHandler, std::move(ui_event)));
 }
 
+void DispatchFocusEventHandler(bool is_focused) {
+  CHECK(ui::PlatformEventSource::GetInstance());
+  static_cast<PlatformEventSourceStarboard*>(
+      ui::PlatformEventSource::GetInstance())
+      ->DispatchFocusEvent(is_focused);
+}
+
+void PlatformEventSourceStarboard::HandleFocusEvent(const SbEvent* event) {
+  if (event->type == kSbEventTypeFocus) {
+    base::SingleThreadTaskRunner::GetCurrentDefault()->PostTask(
+        FROM_HERE, base::BindOnce(&DispatchFocusEventHandler, true));
+  } else if (event->type == kSbEventTypeBlur) {
+    base::SingleThreadTaskRunner::GetCurrentDefault()->PostTask(
+        FROM_HERE, base::BindOnce(&DispatchFocusEventHandler, false));
+  }
```

Comment:
Style nit:

```
  if (event->type != kSbEventTypeFocus && event->type != kSbEventTypeBlur) {
    return;
  }
  const bool is_focused = event->type == kSbEventTypeFocus;
  base::SingleThreadTaskRunner::GetCurrentDefault()->PostTask(
        FROM_HERE, base::BindOnce(&DispatchFocusEventHandler, is_focused));
```
WDYT?
Also would it look readable with DispatchFocusEventHandler() inlined?